doc/config/README.md

```diff
@@ -415,9 +416,9 @@ for certficate update (`KUR`) requests the reprotection mode is always **keep**.
 
 When responding to request messages with successfully verified MAC-based protection,
 the corresponding response messages are protected
-using the same MAC-based algorithm, credentials, and parameters
-(regardless of the configuration related to reprotection or output credentials).
-
+using the same MAC-based algorithm, credentials, and parameters. This behaviour 
+can be modified only by setting `EnforceReprotectMode` to true and `ReprotectMode`
+to **keep** or **strip**.
```

Comment:
```suggestion
using the same MAC-based algorithm, credentials, and parameters. This behaviour 
can be modified only by setting `EnforceReprotectMode` to true.
```
Note that also the **reprotect** value of `ReprotectMode` can lead to different behavior, namely for responses to messages with signature-based protection.